code/modules/mob/mob_helpers.dm

```diff
@@ -478,9 +478,10 @@ GLOBAL_LIST_INIT(intents, list(INTENT_HELP,INTENT_DISARM,INTENT_GRAB,INTENT_HARM
 				lname = "<span class='name'>[lname]</span> "
 			to_chat(M, "<span class='deadsay'>[lname][follow][message]</span>")
 
-/proc/notify_ghosts(message, ghost_sound = null, enter_link = null, title = null, atom/source = null, image/alert_overlay = null, flashwindow = TRUE, action = NOTIFY_JUMP) //Easy notification of ghosts.
+/proc/notify_ghosts(message, ghost_sound = null, enter_link = null, title = null, atom/source = null, image/alert_overlay = null, flashwindow = TRUE, action = NOTIFY_JUMP,role=null) //Easy notification of ghosts.
 	for(var/mob/dead/observer/O in GLOB.player_list)
-		if(O.client)
+
+		if(O.client && role==null?TRUE: (role in O.client.prefs.be_special))
```

Comment:
Generally speaking, conditional operators (`x ? y : z`) are preferred in variable assignments or strings rather than if statements.
```dm
var/example = other_var ? TRUE : FALSE
```
```dm
to_chat(src, "The door is [door.locked ? "" : "un"]locked")
```
<hr>

So this can be simplified to:
```suggestion
		if(O.client && (!role || role in O.client.prefs.be_special))
```